exchanges/stream/websocket_test.go

```diff
@@ -80,15 +168,10 @@ func TestTrafficMonitorTimeout(t *testing.T) {
 	}
 	ws.trafficTimeout = time.Second
 	ws.ShutdownC = make(chan struct{})
-	err = ws.trafficMonitor()
-	if err != nil {
-		t.Fatal(err)
-	}
+	ws.trafficMonitor()
```

Comment:
you could add a check to see if the traffic monitor is running with `w.IsTrafficMonitorRunning()`